.prettierrc.yaml

```diff
@@ -0,0 +1,10 @@
+tabWidth: 2
+useTabs: false
+overrides:
+  - files:
+      - '*.yaml'
```

Comment:
Let's avoid such a deep nesting. Sequences don't need another level of indentation.

Reply:
Let's avoid that formatting debates for now, especially as [the other yaml file](https://github.com/ansible/ansible-language-server/blob/main/.pre-commit-config.yaml) that we already have is using indented sequences, like we also do with Ansible files in general.

If we are about to enforce that, we should do it with a linter, not during code-review. Current change is using the same style as existing files.

Reply:
I allowed that to fast-track the original PR but now it seems wrong to add this format. Especially since there's a far bigger file that is formatted correctly https://github.com/ansible/ansible-language-server/blob/main/.github/workflows/ci.yml. Be consistent with this one, then 🤷‍♂️